### ISSUE TITLE
feature request: create classes from styles

### ISSUE THREAD

**Issue by tobwen** [benboba/svg-slim#25]
## acknowledgement
First of all, thanks for this nice tool. It's by far the best solution on the web. Good work.

## feature request
I've got SVGs with lots of text nodes, which have the same styling. So, it's repeated over and over, which increases the size. Would it be possible to create classes to slim the file down even more? This would save many bytes.

## what it looks like
``` xml
<text style="font-family:Arial;font-size:13.31;font-stretch:Normal;font-weight:400;letter-spacing:0;word-spacing:0" dx="206.4" dy="285.43">70</text>
<text style="font-family:Arial;font-size:13.31;font-stretch:Normal;font-weight:400;letter-spacing:0;word-spacing:0" dx="206.4" dy="253.69">80</text>
<text style="font-family:Arial;font-size:13.31;font-stretch:Normal;font-weight:400;letter-spacing:0;word-spacing:0" dx="206.4" dy="221.95">90</text>
<text style="font-family:Arial;font-size:13.31;font-stretch:Normal;font-weight:400;letter-spacing:0;word-spacing:0" dx="198.7" dy="190.21">100</text>
```

## what it could look like
``` xml
<style type="text/css">
    .textA { font-family: Arial; font-size: 13.31; font-stretch: Normal; font-weight: 400; letter-spacing: 0; word-spacing: 0 }
</style>
<text class="textA" dx="206.4" dy="285.43">70</text>
<text class="textA" dx="206.4" dy="253.69">80</text>
<text class="textA" dx="206.4" dy="221.95">90</text>
<text class="textA" dx="198.7" dy="190.21">100</text>
```

**Comment by benboba** [comment]
Thanks! It's a good idea, I will add this feature later.
I have another idea. This rule can be reversed. If a CSS style is applied to only one element, it can be directly applied to the element.
This feature will be added in version 2.0.1

**Comment by benboba** [comment]
A new rule style-to-class has been created and will be released with version 2.0.1

**Comment by benboba** [comment]
A new rule apply-style has been created and will be released with version 2.0.1
